chsdi/templates/htmlpopup/oev_haltestellen.mako

```diff
@@ -162,23 +149,25 @@ $(document).ready(function() {
 % endif
 </%def>
 
+
 <%def name="extended_info(c, lang)">
 <%
     protocol = request.scheme
+    if lang in ('de', 'rm', 'en'):
+      lang_text = 'de'
+    else:
+      lang_text = 'fr'
```

Comment:
Is it intentionall that it is set to fr?

Reply:
Yes, fr for fr and it and de for all the others...